### ISSUE TITLE
feat(fr24): unlock multiple text labels

### ISSUE THREAD

**Issue by SodaWithoutSparkles** [revanced/revanced-patches#2359]
### Application

flightradar24

### Issue

The data for multiple text labels are there, but hidden behind a paywall

### Patch

Unlock the client-side restrictions on text labels

### Motivation

See #2356

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by ArgX11** [comment]
Yes it would be great to have patch for this app. It's a very useful app but loaded with ads.
